keyboards/uranuma/keymaps/default/keymap.c

```diff
@@ -0,0 +1,65 @@
+#include QMK_KEYBOARD_H
+
+enum layer_number {
+  _BASE,
+  _LOWER,
+  _RAISE,
+};
+
+enum custom_keycodes {
+  KANJI = SAFE_RANGE,
```

Comment:
As a FYI, `KANJI` in this case doesnt actually do anything given there is no `process_record_user` for it.